src/components/Filter.js

```diff
@@ -0,0 +1,30 @@
+import { useRef } from 'react';
+
+const Filter = (props) => {
+
+    // check user input and update filter state to searched term
+    const checkInput = (e) => {
+        console.log(e.target.value);
+        props.filterState(e.target.value);
+    }
+
+    const searchRef = useRef(null);
+    
```

Comment:
maybe you can also add an alert when keyword that is searched doesn't exist and alert user to search again.
I think you mentioned you were going to prevent searches from checking cases (caps/lower cases) and I think that's a great idea!

Reply:
Thank you very much for looking through my code so thoroughly and providing suggestions!! (: I appreciate your help! All suggestions have been implemented!